NDB.Covid19/NDB.Covid19.iOS/Utils/StyleUtil.cs

```diff
@@ -160,11 +160,15 @@ public static void InitUnderlinedLabel(UILabel label, FontType fontType, string
         /// <param name="fontSize"></param>
         /// <param name="maxFontSize"></param>
         /// <param name="alignment"></param>
-        public static void InitLabelWithSpacing(UILabel label, FontType fontType, string rawText, double lineHeight, float fontSize, float maxFontSize)
+        public static void InitLabelWithSpacing(UILabel label, FontType fontType, string rawText, double lineHeight, float fontSize, float maxFontSize, UITextAlignment alignment = UITextAlignment.Left, bool useHyphenation = false)
         {
```

Comment:
Setting UI text alignment to left by default might create problems elsewhere, since we have languages read from right to left such as Arabic, and current GetTextAlignment method sets the alignment based on the language used.

Reply:
Ok, I've changed it.